our-process/policies.md

```diff
@@ -44,16 +44,35 @@ already published on `PyPI` or `conda-forge`.
 
 ### Publication with Journal of Open Source Software (JOSS)
 
-If you have previously published your software package with JOSS, you can still
-submit it to pyOpenSci for review. This provides:
+If you have previously published your software package with the Journal of Open
+Source Software (JOSS), you can still submit it to pyOpenSci for review. This
+provides increased visibility for your package as a vetted tool within the scientific
+Python ecosystem and access to our long-term maintenance support.
 
-- Increased visibility of your package as a vetted tool within the scientific Python
-  ecosystem
-- We will also keep in touch with you as a maintainer to support long-term
-  maintenance. If you need to step down from maintaining your package, we will help
-  find a new maintainer and/or help sunset the tool.
+Since your package has already undergone a JOSS review, we have a specific,
+expedited review process to streamline the submission process and save time.
+
+#### Expedited Review Process
+
+We offer two pathways for packages previously reviewed by JOSS:
+
+1. Fast-Track Review (Editor-Only):
+   Your package is eligible for this fast-track review if it has been published by
+   JOSS within the last year and has not had a major version release (e.g., v1.x to v2.0)
```

Comment:
Maybe mention semantic versioning or clarify "major version" consideration new dependencies & significant API modifications

Reply:
Not sure about others but I think the purpose is to make sure everything in an approved package goes through some kind of review (thinking of an example where a Joss package adds a large feature that's not well documented or very unintuitive).